### PR TITLE
feat(menu): Put list items in submenu, add indentation options

### PR DESCRIPTION
**Pull request by mejo-**
Fixes: #2438
Contributes to: #2836

#### 🖼️ Screenshots

🏚️ Before | 🏡 After
---|---
image | image

| Screencast |
| --- |
| recording1 |

### 🏁 Checklist

- [x] Code is properly formatted (`npm run lint` / `npm run stylelint` / `composer run cs:check`)
- [x] [Sign-off message](https://probot.github.io/apps/dco/) is added to all commits
- [x] [Tests](https://github.com/nextcloud/text#-testing-the-app) (unit, integration and/or end-to-end) passing and the changes are covered with tests

### CODE REVIEW

**Review by jancborchardt — APPROVED** [review]
Very nice! Basically good to go, only 2 remarks:
- [x] A separator line between the list options and indentation options would be useful (as per the suggestion)

Sidenote: This change makes it more obvious that once you decide for ordered/unordered list or checklist, you can not change between them. Any specific reason for that? (But that's a different issue.

**Review by jancborchardt — CHANGES_REQUESTED** [review]
Sorry, actually missed something: It is "indent**ation**", not "indention". :)

File: cypress/e2e/workspace.spec.js [hunk 1]
```diff
@@ -104,15 +104,15 @@ describe('Workspace', function() {
 		})
 	})
 
-	it('creates lists', function() {
+	it.only('creates lists', function() {
```
Comment (juliushaertl) [inline cypress/e2e/workspace.spec.js:107]:
```suggestion
	it('creates lists', function() {
```

**Comment by mejo-** [comment]
> Sidenote: This change makes it more obvious that once you decide for ordered/unordered list or checklist, you can not change between them. Any specific reason for that? (But that's a different issue.

True, but that's another issue tracked in its own issue already. No idea so far, why this is the case.

**Comment by mejo-** [comment]
> - A separator line between the list options and indentation options would be useful (as per the suggestion)
> - (EDIT: Actually both seem to exist, but "indentation" is the more commonly used one. If we want to be short, then we can use "Increase indent" / "Decrease indent".)

Thanks for the review @jancborchardt. Both added now, see the updated screenshot (I didn't update the screencast) :blush:

**Review by juliushaertl — APPROVED** [review]

**Pull request merged**